civihr_employee_portal/views/includes/views_between_operation_dates_filter_handler.inc

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * @file
+ * A filter that allows the between operator to compare dates in 'Y-m-d' format
+ * rather than the default timestamp comparison since absence dates are stored
+ * in 'Y-m-d' format in the db.
+ */
+
+class views_between_operation_dates_filter_handler extends views_handler_filter_date {
+  function op_between($field) {
+    // Use the substitutions to ensure a consistent timestamp.
+    $query_substitutions = views_views_query_substitutions($this->view);
+    $a = intval(strtotime($this->value['min'], $query_substitutions['***CURRENT_TIME***']));
+    $b = intval(strtotime($this->value['max'], $query_substitutions['***CURRENT_TIME***']));
+
+    $a = date('Y-m-d', $a);
```

Comment:
Do you think it would be clearer to use more descriptive variable names?

Reply:
Well, I copied this over from the base class and only added two extra lines of code.
what do you think?

Reply:
Alright, I guess the problem is with the base class then. It might still be nice to change it to `$to` and `$from` or something but I'll leave it to you